File: petties_mobile/lib/data/models/send_otp_response.dart
/// Response when sending OTP for registration
class SendOtpResponse {
  final String message;
  final String email;
  final int expiryMinutes;
  final int resendCooldownSeconds;

  SendOtpResponse({
    required this.message,
    required this.email,
    required this.expiryMinutes,
    required this.resendCooldownSeconds,
  });

  factory SendOtpResponse.fromJson(Map<String, dynamic> json) {
    return SendOtpResponse(
      message: json['message'] ?? '',
      email: json['email'] ?? '',
      expiryMinutes: json['expiryMinutes'] ?? 5,
      resendCooldownSeconds: json['resendCooldownSeconds'] ?? 60,
    );
  }

  Map<String, dynamic> toJson() {
    return {
      'message': message,
      'email': email,
      'expiryMinutes': expiryMinutes,
      'resendCooldownSeconds': resendCooldownSeconds,
    };
  }
}

File: backend-spring/petties/src/main/java/com/petties/petties/dto/auth/SendOtpResponse.java
package com.petties.petties.dto.auth;

import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;

/**
 * Response DTO khi gửi OTP thành công
 */
@Getter
@Setter
@NoArgsConstructor
@AllArgsConstructor
@Builder
public class SendOtpResponse {

    private String message;
    private String email;
    private int expiryMinutes;
    private int resendCooldownSeconds;
}
